syntax = "proto3";

option csharp_namespace = "PiecykPolHurt.DataImport";

package productUpdate;

// The greeting service definition.
service ProductUpdater {
  // Sends a greeting
  rpc UpdateProduct (ProductUpdateRequest) returns (ProductUpdateResponse);
}

// The request message containing the user's name.
message ProductUpdateRequest {
  ProductType type = 1;
  int32 quantity = 2;
  string branch = 3;
}

// The response message containing the greetings.
message ProductUpdateResponse {
  string message = 1;
}

message ProductType {
  enum PRODUCT_TYPE {
    ORZECH = 0;
    KOSTKA = 1;
    GROSZEK = 2;
    MIAL = 3;
  }
  PRODUCT_TYPE productType = 1;
}
